### PR TITLE
add AASIST

### PR DESCRIPTION
**Pull request by yzyouzhang**

### CODE REVIEW

File: versa/utterance_metrics/asvspoof_score.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+# Author: You (Neil) Zhang
+
+## This is to evalute the generated speech with a deepfake detection model
+# We include the AASIST model trained on ASVspoof 2019 LA dataset to 
+# output the confidence score of whether the speech input is a deepfake
+# Please refer to https://github.com/clovaai/aasist for more details
+
+
+## cloning the repo into the checkpoint folder
+## git clone git@github.com:clovaai/aasist checkpoints/aasist
```
Comment (ftshijt) [inline versa/utterance_metrics/asvspoof_score.py:15]: We may add an installer file to `tools` for this option?

Comment (yzyouzhang) [reply versa/utterance_metrics/asvspoof_score.py]: Updated.

**Review by ftshijt — COMMENTED** [review]
Many thanks! For a quick follow up, could you please:
- Add test function in `test/test_asvspoof.py`
- Add example config in `egs/separate_metric/asvspoof.yaml`

**Comment by ftshijt** [comment]
Many thanks!

**Pull request merged**